### PR TITLE
Deprecate Persona as authentication provider

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: feedthefox/base/templates/jinja2/base.html [hunk 1]
```diff
@@ -159,7 +150,6 @@
 
     <script src="{{ static('js/jquery-2.1.1.min.js') }}"></script>
     <script src="{{ static('js/bootstrap.min.js') }}"></script>
-    <script src="https://login.persona.org/include.js"></script>
     <script src="{{ static('js/auth.js') }}"></script>
```
Comment (akatsoulas) [inline feedthefox/base/templates/jinja2/base.html]: Probably you need to remove `auth.js` both from here and the project in general.

File: feedthefox/base/templates/jinja2/base.html [hunk 2]
```diff
@@ -81,18 +81,9 @@
                       <li><a href="{{ url('account_logout') }}"><i class="fa fa-sign-out"></i> Logout</a></li>
                     {% else %}
                       <li>
-                        <a href="javascript:allauth.persona.login('', 'login')"><i class="fa fa-user"></i> Persona</a>
-                      </li>
-                      <li>
                         <a href="{{ provider_login_url('github') }}"><i class="fa fa-github"></i> Github</a>
                       </li>
                     {% endif %}
-                    <form id="_persona_login" method="post" action="{{ url('persona_login') }}">
```
Comment (akatsoulas) [inline feedthefox/base/templates/jinja2/base.html:90]: Please also remove persona as provider from allauth in `settings.py`.

**Review by akatsoulas — APPROVED** [review]
r+wc. Code looks OK but I haven't tested this.

**Comment by akatsoulas** [comment]
Please also upgrade django to version 1.8.15 in the first commit.

**Comment by brianking** [comment]
Wasting your time here, this site should be shut down - #229

**Comment by johngian** [comment]
@brianking This what we were planning, apparently there are some links in bedrock that point to the portal and we didn't want to leave them broken.

**Pull request merged**